dubbo-dependencies-bom/pom.xml

```diff
@@ -136,10 +136,10 @@
         <!-- Log libs -->
         <slf4j_version>1.7.25</slf4j_version>
         <jcl_version>1.2</jcl_version>
-        <log4j_version>1.2.16</log4j_version>
-        <logback_version>1.2.2</logback_version>
+        <log4j_version>1.2.17</log4j_version>
+        <logback_version>1.2.11</logback_version>
         <!-- Fix the bug of log4j refer:https://github.com/apache/logging-log4j2/pull/608 -->
-        <log4j2_version>2.17.0</log4j2_version>
```

Comment:
Thank you for your contribution, there are still some areas that need to be upgraded, such as `dubbo-spring-boot` module , etc.